routes/resources.js

```diff
@@ -10,17 +10,17 @@ var Schemas = require('../generators/schemas');
 
 module.exports = function (app, model, Implementation, integrator, opts) {
   var modelName = Implementation.getModelName(model);
-  var schema = Schemas.schemas[Implementation.getModelName(model)];
+  var schema = Schemas.schemas[modelName];
 
   function injectSmartFields(record) {
     return P.each(schema.fields, function (field) {
       if (!record[field.field]) {
         if (field.get || field.value) {
           if (field.value) {
-            logger.warn('DEPRECATION WARNING: Smart Field value method is ' +
-              'deprecated. Please use get method instead.');
+            logger.warn('DEPRECATION WARNING: In your ' + field.field +
+              ' Collection, Smart Field value method is deprecated. ' +
```

Comment:
I am not sure that `field.field` is always related to a Collection (fullname is not a Collection for example).